### PR TITLE
Complete the League Checklist

### PR DESCRIPTION
**Pull request by kylestev**
fixes the following items:

* Switch to ```League\``` namespace
* `PSR-2` compliance
* `PSR-4` compliance
* Added/updated `README.md`, `CONTRIBUTING.md`, `CHANGELOG.md` to follow thephpleague/skeleton

closes zackkitzmiller/tiny-php#9

### CODE REVIEW

**Comment by zackkitzmiller** [comment]
@philsturgeon, take a look?

**Comment by zackkitzmiller** [comment]
@philsturgeon, take a look?

**Comment by zackkitzmiller** [comment]
Looks good, once Phil signs off, we're done here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Changelog
+
+All Notable changes to `league/tiny` will be documented in this file
+
+## 2015-06-18
+
+### Added
```
Comment (philsturgeon) [inline CHANGELOG.md]: You dont need to list nothing changes, just omit them.

**Comment by philsturgeon** [comment]
Looks good to me! 

Here is the [package submission process](https://github.com/thephpleague/thephpleague.github.io/wiki/Package-Submission)

Dunno if you wanna project lead zack, or @kylestev does, or if we want to find somebody. Either way posting on the ML would be a good idea. :)

**Comment by zackkitzmiller** [comment]
@philsturgeon I'll lead.

**Comment by kylestev** [comment]
:beers:

**Pull request merged**